public class Question4 {
    public static void main(String[] args) {
        int result = 0;
        try {
            result = 10/0;
        } catch (Throwable e) {
            System.out.println("Caught exception: " + e.getMessage());
        }
        System.out.println(result);
    }
}
